cypress/integration/basic-auth.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { datasetPage } from "../page-objects/app/datasets-page";
+
+describe("Basic browser auth", () => {
+
+  it("should login into webApp", () => {
+    datasetPage.visit();
```

Comment:
Replace at package.json this line
`"lint": "eslint -c .eslintrc.js --ext js,ts,tsx src",`
with
`"lint": "eslint -c .eslintrc.js --ext js,ts,tsx src cypress",`
and lint project with `yarn lint:fix`

Automatically not resolved features should be resolved manually